book/emails.rst

```diff
@@ -0,0 +1,320 @@
+.. index::
+    :single: E-Mails
+
+Working with E-Mails
+====================
+
+Communicating with customers is an important part of every business. In the Oro Platform, an e-mail
+address is represented by the :class:`Oro\\Bundle\\EmailBundle\\Entity\\Email` activity.
+
+E-Mail Templates
+----------------
+
+When creating e-mails, your users often want to reuse texts they have written some time before for
+a certain purpose. For this, they can create templates and reuse them as needed. Besides letting
+the users create and manage templates through the UI, every bundle can provider their own e-mail
+templates.
+
+Providing e-mail templates is nothing more than creating Twig templates for the desired format
+(namely HTML or plaintext) in which you can use some pre-defined placeholders to define template
+metadata. Then you create a data fixture that implements the ``AbstractEmailFixture`` class. This
+class provides a ``getEmailsDir`` method which will point to the directory that contains your
+templates:
+
+.. code-block:: php
+    :linenos:
+
+    // src/Acme/DemoBundle/DataFixtures/ORM/EmailTemplatesFixture.php
+    namespace Acme\DemoBundle\DataFixtures\ORM;
+
+    use Oro\Bundle\EmailBundle\DataFixtures\ORM\AbstractEmailFixture;
+
+    class EmailTemplatesFixture extends AbstractEmailFixture
+    {
+        public function getEmailsDir()
+        {
+            return __DIR__.'/../../Resources/emails';
```

Comment:
This is more preferable implementation of getEmailsDir method's body:
        return $this->container
            ->get('kernel')
            ->locateResource('@AcmeDemoBundle/Migrations/Data/ORM/emails');